tests/integration/cloudformation/resources/test_apigateway.py

```diff
@@ -438,3 +442,111 @@ def test_api_gateway_with_policy_as_dict(deploy_cfn_template, snapshot, aws_clie
     rest_api["policy"] = json.loads(policy)
 
     snapshot.match("rest-api", rest_api)
+
+
+@pytest.mark.aws_validated
+@pytest.mark.skip_snapshot_verify(
+    paths=[
+        "$.put-ssm-param.Tier",
+        "$.get-resources.items..resourceMethods.GET",
+        "$.get-resources.items..resourceMethods.OPTIONS",
+        "$..methodIntegration.cacheNamespace",
+        "$.get-authorizers.items..authorizerResultTtlInSeconds",
+    ]
+)
+def test_rest_api_serverless_ref_resolving(
+    deploy_cfn_template, snapshot, aws_client, create_parameter, create_lambda_function
+):
+    template = """
+    AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Any particular reason keeping the template here inline instead of a separate file?

Reply:
The template was really small in the beginning, but now that it became bigger I've pulled it into a file, thanks!